pkg/render/fluentd.go

```diff
@@ -962,20 +1016,24 @@ func (c *fluentdComponent) fluentdClusterRole() *rbacv1.ClusterRole {
 				APIGroups: []string{"linseed.tigera.io"},
 				Resources: []string{
 					"flowlogs",
-					"kube_auditlogs",
-					"ee_auditlogs",
-					"dnslogs",
-					"l7logs",
-					"events",
-					"bgplogs",
-					"waflogs",
-					"runtimereports",
 				},
 				Verbs: []string{"create"},
 			},
 		},
 	}
 
+	if c.cfg.EnterpriseVariant {
+		role.Rules[0].Resources = append(role.Rules[0].Resources, []string{
```

Comment:
Let's not do this based on `Rules[0]` - that is likely to introduce a bug in the future when someone modifies the rules.

Please build the rule first, and then add it to the Rules list.

```go
linseedRule := rbacv1.PolicyRule{ ...}
if enterprise{
  linseedRule.Resources = ...
}

role.Rules = append(role.Rules, linseedRule)
```